node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@herp-inc/eslint-config-node",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
The `version` property is automatically updated when the maintainer runs `$ yarn version` before publishing the package, so please let it unchanged.